.pre-commit-config.yaml

```diff
@@ -1,45 +1,53 @@
 default_language_version:
     python: python3.8
-files: "\
-        ote_sdk"
+
 repos:
 -   repo: https://github.com/PyCQA/isort
     rev: '5.9.2'
     hooks:
     -   id: isort
         args: ["--settings", ".isort.cfg"]
+        files: '^(ote_sdk|ote_cli|external)/.*\.py'
 
 -   repo: https://github.com/psf/black
     rev: 21.7b0
     hooks:
     -   id: black
         name: 'black'
-        files: '^ote_sdk/.*\.py'
+        files: '^(ote_sdk|ote_cli|external)/.*\.py'
 
 -   repo: https://github.com/PyCQA/flake8
     rev: '3.9.2'
     hooks:
     -   id: flake8
         name: 'flake8'
-        files: '^ote_sdk/.*\.py'
+        files: '^(ote_sdk|ote_cli|external)/.*\.py'
         args: ["--config", ".flake8", "--max-complexity", "20"]
         exclude: ".*/protobuf"
 
 -   repo: https://github.com/pre-commit/mirrors-mypy
     rev: 'v0.812'
     hooks:
     -   id: mypy
-        name: 'mypy'
+        name: 'mypy (ote_sdk)'
         files: '^ote_sdk/.*\.py'
         args: ["--config-file=ote_sdk/.mypy.ini"]
         additional_dependencies: [numpy==1.19.5, types-PyYAML, attrs==21.2.*, types-requests, types-Deprecated, types-docutils, types_futures, types-python-dateutil]
+    -   id: mypy
+        name: 'mypy (ote_cli)'
+        files: '^ote_cli/.*\.py'
+        args: ["--config-file=ote_sdk/.mypy.ini"]
+    -   id: mypy
+        name: 'mypy (external)'
+        files: '^external/.*\.py'
+        args: ["--config-file=ote_sdk/.mypy.ini"]
```

Comment:
Does it make sense to also set aliases for these hooks? Or is it possible to run them separately now?

Reply:
Added